api/crudFunctions/weeklyPicks.tsx

```diff
@@ -0,0 +1,36 @@
+//globally accessible env variable
+declare global {
+    namespace NodeJS {
+        export interface ProcessEnv {
+            [key: string]: string | undefined;
+            PROJECT_ID: string;
+            API_KEY: string;
+        }
+    }
+}
+
+export async function getWeeklyPicks() {
```

Comment:
This is the wrong location for this function. We wanted you to build it inside https://github.com/LetsGetTechnical/gridiron-survivor/blob/develop/api/authFunctions.ts. Keep all model calls in 1 place for now.